src/modules/visitor-space/components/MaintainerFilterForm/MaintainerFilterForm.const.ts

```diff
@@ -0,0 +1,15 @@
+import { ArrayParam } from 'use-query-params';
+import { array, object, SchemaOf, string } from 'yup';
+
+import { VisitorSpaceFilterId } from '../../types';
+
+import { MaintainerFilterFormState } from './MaintainerFilterForm.types';
+
+export const MAINTAINER_FILTER_FORM_SCHEMA = (): SchemaOf<MaintainerFilterFormState> =>
+	object({
+		maintainers: array(string().required()),
```

Comment:
```suggestion
		[VisitorSpaceFilterId.Maintainers]: array(string().required()),
```

Reply:
Dit werkt niet aangezien de value van `VisitorSpaceFilterId.Maintainers` `aanbieders` is en niet `maintainers`. Idem op de andere plaatsen ;)

Als we dit op een propere manier willen oplossen zouden we een Record moeten toevoegen waarbij we die waarden kunnen linken aan elkaar. Als we dit zouden doen, zou dit doorgetrokken moeten worden naar alle filters die gebruikt worden binnen de zoekpagina.